### ISSUE TITLE
ability to expose methods on web component

### ISSUE THREAD

**Issue by rburgst** [ryansolid/component-register#14]
Hi, is it possible to expose custom methods on the final web component object? I would need a way to expose some api from the component (think of a player `play()`, `pause()`).

**Comment by ryansolid** [comment]
Yeah.. the second argument passes element in which is the host DOM element. You can just append stuff to it.

Like in the get started in the readme it shows setting the HTML. But you could also define properties on it.
```js
import { register } from 'component-register';

register('my-element')((props, { element }) =>
  element.play = () => // make it play
  element.pause = () => // make it pause
  element.renderRoot.innerHTML = 'Hello World'
)
```

**Comment by rburgst** [comment]
it would be cool to have an example on how to integrate that with a SolidJS web component.
